runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectRunner.java

```diff
@@ -274,12 +268,12 @@ public MetricResults metrics() {
     }
 
     /**
-     * Blocks until the {@link Pipeline} execution represented by this
-     * {@link DirectPipelineResult} is complete, returning the terminal state.
+     * Blocks until the {@link Pipeline} execution represented by this {@link DirectPipelineResult}
```

Comment:
I'm not sure how much I want this class to be package private, because it does show up in the generic type of the `DirectRunner`.

FWIW this documentation exists to specify the rethrowing behavior. Cut down, and only added that additional paragraph.